recipe/meta.yaml

```diff
@@ -11,14 +11,13 @@ source:
   sha256: e2c71babfd18a8e69542dd7e9ca018f9caa438094001a58e6bc4d8c999bf0d07
 
 build:
-  number: 0
-  noarch: generic
+  number: 1
   skip: true # [osx or (linux and s390x)]
 
 requirements:
   run:
-    - __linux  # [linux]
-    - __win    # [win]
+    #- __linux  # [linux]
+    #- __win    # [win]
```

Comment:
This can be completely removed as it's conda-forge stuff for noarch packages

Reply:
I've followed the pattern of commenting out rather than removing lines, to keep a clearer visual record of what's changed. I'd rather stick to the pattern followed in the other feedstocks.